source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -171,31 +171,45 @@ If you are using CRI-O and `flannel` and you want to use Kata Containers, edit t
     [crio.runtime]
     manage_network_ns_lifecycle = true
 
-**Notes about Weave Net add-on**
+.. TODO: Should Weave and Calico be mentioned below, or is Flannel enought?
 
-If you choose the `Weave Net` add-on, then you must make the following
-changes because it installs itself in the :file:`/opt/cni/bin` directory.
+Create a symlink for the network overlays:
 
-For using CRI-O and `Weave Net`, you must complete the following
-steps.
+.. code-block:: bash
+
+   sudo ln -s /usr/libexec/cni /opt/bin/cni
+
+.. note::
+
+   |CL| installs CNI plugins that are part of the `cloud-native-basic` bundle to :file:`/usr/libexec/cni`. The directory is required because `swupd verify` uses it, if necessary, to repair a system to a known good state.
+
+.. TODO: Verify whether to omit this section per mythi, eadamsintel.
+
+.. **Notes about Weave Net add-on**
+
+.. If you choose the `Weave Net` add-on, then you must make the following
+.. changes because it installs itself in the :file:`/opt/cni/bin` directory.
+
+.. For using CRI-O and `Weave Net`, you must complete the following
+.. steps.
 
-#.  Edit the :file:`/etc/crio/crio.conf` file to change `plugin_dir` from:
+.. #.  Edit the :file:`/etc/crio/crio.conf` file to change `plugin_dir` from:
 
-    ..  code-block:: bash
+..     ..  code-block:: bash
 
-        plugin_dir = "/usr/libexec/cni/"
+..         plugin_dir = "/usr/libexec/cni/"
 
-    to:
+..     to:
 
-    ..  code-block:: bash
+..     ..  code-block:: bash
 
-        plugin_dir = "/opt/cni/bin"
+..         plugin_dir = "/opt/cni/bin"
 
-#.  Add the `loopback` CNI plugin to the plugin path with the command:
+.. #.  Add the `loopback` CNI plugin to the plugin path with the command:
 
-    ..  code-block:: bash
+..     ..  code-block:: bash
 
-        sudo ln -s /usr/libexec/cni/loopback /opt/cni/bin/loopback
+..         sudo ln -s /usr/libexec/cni/loopback /opt/cni/bin/loopback
```

Comment:
This is the reason we need container-basic as that provides the CNI plugins today (I think).
Longer term we should just add the CNI plugins to container-native-basic.

@bryteise can we add CNI plugins to container-native-basic

It is preferrable not to have container-basic as docker adds its own iptable rules.
No having docker for k8s, would make the system cleaner

Reply:
I've submitted a patch for review adding CNI to cloud-native-basic.

Reply:
What was wrong with my patch sent a week ago? ;)